README.md

```diff
@@ -23,7 +23,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.remoteflags</groupId>
   <artifactId>remoteflags-java-client</artifactId>
-  <version>1.0.2</version>
+  <version>1.0.7</version>
```

Comment:
Just to double check, this is to go hand in hand with node version right? Not sure we should, maybe we will update dependencies in Java and note node. 

However the major and minor makes sense, having said that, this can be a 1.1.0